pkg/controllers/management/aks/aks_cluster_handler.go

```diff
@@ -0,0 +1,696 @@
+package aks
```

Comment:
You can import `"github.com/rancher/rancher/pkg/controllers/management/clusteroperator"` and reuse a lot of the constants, types, and methods from there for this.